src/popover.scss

```diff
@@ -6,24 +6,44 @@ $block: #{$fd-namespace}-popover;
 
 .#{$block} {
 
-  $fd-popover-top-position: 120% !default;
+  $fd-popover-top-position: calc(100% + 0.5rem) !default;
```

Comment:
you actually don't need calc when using sass it does it for you

Reply:
Hmm not really because we are using 100%.

Here is a good explanation from Stackoverflow:
"SASS height: 100px / 2; will compile to height: 50px; <- and this is what the browser will see always no matter what.

CSS width: calc(100% - 20px) will always be this even on the browser, and at that point the browser will do the calculations depending on what that 100% looks like."

https://stackoverflow.com/questions/52869421/sass-calc-vs-normal-calculation